controllers/adminController.js

```diff
@@ -0,0 +1,61 @@
+const userRoleManager = require('../lib/modelManagers/userRoleManager');
+const userModelManager = require('../lib/modelManagers/usermodel');
+const { successResponse, failureResponse, serverFailure } = require('../lib/utils/messageHandler');
+const {
+  ROLE_CREATED,
+  RESOURCE_CREATED_CODE,
+  ROLE_ALREADY_EXIST,
+  CONFLICT_CODE,
+  NOT_FOUND_CODE
+} = require('../constants/index');
+
+/**
+ *
+ *
+ * @param {*} req
+ * @param {*} res
+ * @returns {*} *
+ */
+async function createRoleController(req, res) {
+  const { role } = req.body;
+  try {
+    const record = await userRoleManager.findRole(req.body.role);
+    if (record) {
+      return failureResponse(res, ROLE_ALREADY_EXIST, CONFLICT_CODE);
+    }
+    const newRole = await userRoleManager.createRole(role);
+    return successResponse(res, ROLE_CREATED, RESOURCE_CREATED_CODE, newRole);
+  } catch (error) {
+    return serverFailure(res);
+  }
+}
+
+/**
+ *
+ *
+ * @param {*} req
+ * @param {*} res
+ * @returns {*} *
+ */
+async function createUserRoleController(req, res) {
```

Comment:
The controller for adding roles to a user should be in the user controller. I also expect that there should be a controller to fetch a user's role, update and remove a particular role from a user.

Reply:
I have added the controllers